vllm/v1/core/sched/scheduler.py

```diff
@@ -751,18 +774,14 @@ def update_from_output(
                         new_logprobs=new_logprobs,
                         new_prompt_logprobs_tensors=prompt_logprobs_tensors,
                         stop_reason=request.stop_reason,
-                        events=request.take_events()))
+                        events=request.take_events(),
+                        kv_transfer_params=kv_transfer_params,
+                    ))
+
             else:
                 # Invariant: EngineCore returns no partial prefill outputs.
                 assert not prompt_logprobs_tensors
 
-            if self.connector is not None and request.do_remote_decode:
-                stopped = True
-                self.waiting_to_send_KV_req_ids.add(req_id)
```

Comment:
We still need this

Reply:
why cant we trigger the sending during the model forward pass?

Reply:
but I updated above to put into `waiting_to_send_KV_req_ids` instead `sending_KV_req_ids`

Reply:
ok that works. (the whole reason for waiting_to_send_KV_req_ids is to make it easy to start sending during the forward pass)